src/components/dashboard/Claim.js

```diff
@@ -111,7 +129,15 @@ const Claim = props => {
     const [claimedToday, nextClaimDate] = await Promise.all([
       wrappedGoodWallet.getAmountAndQuantityClaimedToday(),
       wrappedGoodWallet.getNextClaimTime(),
-    ])
+    ]).catch(e => {
+      log.error('gatherStats failed', e.message, e)
+      showErrorDialog('Something unexpected happened', '', {
```

Comment:
@StanislavShevchenko please change this error message to: "The error happens in blockchain request, please try again later"